package.json

```diff
@@ -68,7 +687,7 @@
         "controlType": "textbox"
```

Comment:
This is not user friendly. It should be a search like the international weather applet